recipe/meta.yaml

```diff
@@ -12,12 +12,12 @@ package:
 source:
   fn: {{ name }}-{{ version }}.{{ bundle }}
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  {{ hash_type }}: {{ hash_val }}
+  {{ hash_type }}: {{ hash }}
 
 build:
+  noarch: python
   entry_points:
     - jwt = jwt.__main__:main
```

Comment:
Not sure how this didn't already cause problems, but it seems this should be called [`pyjwt`]( https://github.com/jpadilla/pyjwt/commit/5c5fab1282c21a4a37dfebae54b2d4bc1ea7f037#diff-2eeaed663bd0d25b7e608891384b7298R81 ). Seems this has been the case since `1.5.0`.

Reply:
Also the `commands` test below will need to be updated to use `pyjwt` after this change.